gamification/process/update.py

```diff
@@ -135,6 +136,29 @@ def get_participant_objects():
     return participants
 
 
+def update_participants_data_with_meta_review(participants):
+    """
+    Update participants based on meta-review
+
+    This method updates every participant based on the meta-review
+    recieved or given. If the meta-review is completed then it calls
+    'Completed Meta-Review' activity.
+    """
+    pos_in_meta_review = meta_review_participants.objects.get(
+                        pos_in=participants.pos_in)
```

Comment:
This part makes me a bit confused. Does this ever work? Apparently `participants` come from `gamification.models`, which doesn't have `pos_in` field.

`meta_review_participants.objects.get(pos_in=some number)` doesn't make sense to me either. This returns an object whose `pos_in` field = `some number`.